README.md

```diff
@@ -8,7 +8,7 @@ A safe, extensible ORM and Query Builder for Rust
 [![Gitter](https://badges.gitter.im/diesel-rs/diesel.svg)](https://gitter.im/diesel-rs/diesel?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![Crates.io](https://img.shields.io/crates/v/diesel.svg)](https://crates.io/crates/diesel)
 
-[Documentation](http://docs.diesel.rs)
+API Documentation: [latest release](http://docs.rs/diesel) – [master branch](http://docs.diesel.rs)
```

Comment:
Can this be changed to https ? See #1293 too

Reply:
Good point! I made all the links I added/changed https.